pkg/kv/kvclient/rangefeed/config.go

```diff
@@ -37,24 +37,26 @@ type config struct {
 	extraPProfLabels     []string
 }
 
-type scanConfig struct {
-	// scanParallelism controls the number of concurrent scan requests
+// ScanConfig is used to configure rangefeed scans.
+type ScanConfig struct {
+	// ScanParallelism controls the number of concurrent scan requests
 	// that can be issued.  If unspecified, only 1 scan request at a time is issued.
-	scanParallelism func() int
+	ScanParallelism func() int
```

Comment:
It sorta feels like if we're going to stick with the functional options then we should expose the values via methods.